docs/src/main/asciidoc/datasource.adoc

```diff
@@ -410,18 +394,19 @@ quarkus.datasource.inventory.jdbc.url=jdbc:h2:mem:inventory
 quarkus.datasource.inventory.jdbc.max-size=12
 ----
 
-Notice there is an extra bit in the key.
+Notice there is an extra section in the configuration key.
 The syntax is as follows: `quarkus.datasource.[optional name.][datasource property]`.
 
-NOTE: Even when only one database extension is installed, named databases need to specify at least one build time
-property so that Quarkus knows they exist. Generally this will be the `db-kind` property, although you can also
+NOTE: Even when only one database extension is installed, named databases need to specify at least one build-time property so that Quarkus can detect them.
+Generally, this is the `db-kind` property, but you can also
 specify Dev Services properties to create named datasources (covered later in this guide).
```

Comment:
Dev services are not covered in this guide, but in another one (`xref:databases-dev-services.adoc`).